src/pretalx/schedule/models/schedule.py

```diff
@@ -143,9 +143,60 @@ def changes(self):
             result['action'] = 'create'
             return result
 
+        # ******************************************
+        # build list of all new talks without link to schedule
+        # from pretalx.submission.models import Submission
+        # qs = Submission.objects.filter(
+        #     id__in=self.scheduled_talks.values_list('submission', flat=True)
+        # )
+        # print('qs')
+        # for slot in qs:
+        #     print(slot)
+        # #
+        #
+        # print('****')
+
+        new_slots_qs = self.talks.select_related(
+            'submission', 'submission__event', 'room'
+        ).all().filter(
+            is_visible=True,
+            room__isnull=False,
+            # start__isnull=False,
```

Comment:
Leaving `start__isnull=False` seems like a good idea to make sure no accidentally unscheduled talks appear.

Reply:
just for the record:
it is a good idea yes.
but the `is_visible=True` is not correct at all!
(this will be set by the freeze function - and so i would filter out  talks that are not jet scheduled after a release..)